Training/configs/training_phase2.yaml

```diff
@@ -3,13 +3,12 @@ Setup:
     iso_cone             :  0.5
     n_inner_cells        :  11
     inner_cell_size      :  0.02
-    n_outer_cells        :  21
-    outer_cell_size      :  0.05
+    n_outer_cells        :  25
+    outer_cell_size      :  0.042
```

Comment:
Do we want to have `n_outer_cells=25` as the baseline for Phase 2? If yes, could you add to the PR description a link to the slides that justify the increase of the number of cells?